src/plugins/data_source/server/plugin.ts

```diff
@@ -32,5 +49,20 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
     return {};
   }
 
-  public stop() {}
+  public stop() {
+    this.dataSourceService!.stop();
+  }
+
+  private createRouteHandlerContext = (
+    core: CoreSetup
+  ): IContextProvider<RequestHandler<unknown, unknown, unknown>, 'data_source'> => {
+    return async (context, req) => {
+      const [{ savedObjects }] = await core.getStartServices();
```

Comment:
you can use `context.savedObjects.client`

Reply:
yeah you are right, let me fix that